sdk/core/perf/inc/azure/perf/dynamic_test_options.hpp

```diff
@@ -72,5 +76,7 @@ namespace Azure { namespace Perf {
     {
       return m_results[optionName].as<T>();
     }
+
+    bool ShouldSkipInitial() { return m_skipInitial; }
```

Comment:
Skip initial ... what? Skip initial run? (Should the name be `SkipInitialRun()`) Not obvious from the name. From the code in the same PR, I kinda of can see that we skip one run when this is true. But again, why, what for are we skipping it? Why do we want to skip the initial run?

Please add a comment to this method explaining the intended purpose.